README.rst

```diff
@@ -75,197 +77,186 @@ Installation
+* ``debug``: works the same as verbose, but temporary files and
+  directories aren't removed during command execution.
+* ``quiet``: hides all logs, only displays error messages.
+
+Application lifecycle
+~~~~~~~~~~~~~~~~~~~~~
 
 In a nutshell:
 
-1. `Create <Creating an application from template_>`_
-   an application (e.g. ``myapp``) from template:
+1. :ref:`Create an application <cartridge-cli-creating_an_application_from_template>`
+   (for example, ``myapp``) from a template:
 
    .. code-block:: bash
 
        cartridge create --name myapp
        cd ./myapp
 
-2. `Build <Building an application_>`_ the application
+2. :ref:`Build the application <cartridge-cli-building-the-application>`
    for local development and testing:
 
    .. code-block:: bash
 
        cartridge build
 
-3. `Run <Starting/stopping an application locally_>`_
-   instances locally:
+3. :ref:`Run instances locally <cartridge-cli-starting-the-application-locally>`:
 
    .. code-block:: bash
 
        cartridge start
        cartridge stop
 
-4. `Pack <Packing an application_>`_ the application into
-   a distributable (e.g. into an RPM package):
+4. :ref:`Pack the application <cartridge-cli-packaging-the-application>`
+   into a distributable (like an RPM package):
 
    .. code-block:: bash
 
        cartridge pack rpm
 
 .. _cartridge-cli-creating_an_application_from_template:
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Creating an application from template
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Creating an application from a template
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To create an application from the Cartridge template, say this in any directory:
+To create an application from a Cartridge template, run ``cartridge create`` in any directory:
 
-.. code-block:: bash
+..  code-block:: bash
 
     cartridge create [PATH] [flags]
 
-The following options (``[flags]``) are supported:
-
-.. // Please, update the doc in cli/commands on updating this section
+Supported options (``[flags]``):
 
-* ``--name string`` is an application name.
+.. // Please update the doc in cli/commands on updating this section
 
-* ``--from DIR`` is a path to the application template (see details below).
+*   ``--name string``: application name.
 
-* ``--template string`` is a name of application template to be used.
-  Currently only ``cartridge`` template is supported.
+*   ``--from DIR``: path to the application template. For more about templates, see below.
 
-Application is created in the ``<path>/<app-name>/`` directory.
+*   ``--template string``: name of the application template to be used.
+    Only the template ``cartridge`` is supported at the time.
 
-By default, ``cartridge`` template is used.
-It contains a simple Cartridge application with:
+Your application will appear in the ``<path>/<app-name>/`` directory.
```

Comment:
It is named just `<app_name>/` below. I'm for consistency (I would prefer just `<app_name>/`).